main.py

```diff
@@ -189,9 +189,39 @@ def _cleanup_repos(repos: list):
     run("rm -rf repos".split())
 
 
+def _find_ignored_files(repo: str) -> list:
+    # Store already excluded dirs to avoid unneccessarily iterating a bunch of already excluded paths
+    excluded_dirs = []
+    output = []
+
+    for root, dirs, files in os.walk(os.path.join('repos', repo)):
+        if [1 for excluded in excluded_dirs if root.replace(os.path.join('repos', repo), '').strip('/').startswith(excluded)]:
```

Comment:
I think I kinda understand what this does, but maybe... add a comment for the future? c:

Reply:
Should be solved with 2c9c60b